src/stories/Library/rich-text/rich-text.scss

```diff
@@ -75,6 +75,25 @@ $_max-width-rich-text: $block-max-width__small;
     }
   }
 
+  blockquote {
+    $_quote-indicator-width: 10px;
```

Comment:
:+1: I liked this use of a variable.

I wonder if there is a reason why you went with `10px` instead of e.g. `8px` (`$s-sm`) which is already a size we use in the design.